components/steps/Profile.tsx

```diff
@@ -112,7 +112,7 @@ const ProfileStep: React.FunctionComponent<Props> = props => {
     <Flex direction="column">
       <Form
         onSubmit={e => {
-          if (userResume) {
+          if (userResume && userResume.files[0]) {
```

Comment:
Adding checks if the user has selected a file, otherwise it'll try to upload an `undefined` resume

Reply:
I'm dumb, thanks